### PR TITLE
Add STROKE panel to clinical master list

### LINKED ISSUE #2187: Customer request: add Moyamoya panel to clinical master list
## Description

`cust002` wants Moyamoya (STROKE) on the clinical master list.

## Suggested solution

Add it to the master list.

## This can be closed when

Panel STROKE is listed as part of at least a MIP dry run start.

### PR DESCRIPTION
**Pull request by dnil**
## Description

Fix #2187 - add Moyamoya STROKE panel to clinical master list

### Added

-

### Changed

- add Moyamoya STROKE panel to clinical master list

### Fixed

-

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [x] Do attempt a case order with a master list panel with cg and note the new panel on the clinical list being fetched from Scout, e.g.
```
cg workflow mip-dna panel -d justhusky
```

### Expected test outcome

- [x] Check that it worked
- [x] Take a screenshot and attach or copy/paste the output.

## Review

- [x] Tests executed by DN
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by ChrOertlin — APPROVED** [review]
A constant "STROKE" has been added tot he GenePanelMasterList, looks good!

File: cg/constants/gene_panel.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ class GenePanelMasterList(StrEnum):
     PANELAPP_GREEN: str = "PANELAPP-GREEN"
```
Comment (ChrOertlin) [inline cg/constants/gene_panel.py:37]: Just a minor thing, there are a few panel names out of alphabetical order starting with PEDHEP, would be nice to fix that. But not strictly necessary

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by dnil** [comment]
Deployed to prod:
```
Log deploy... done.
cg, version 37.0.21
```

**Pull request merged**